photo_points/Services/AdminReviewServices.cs

```diff
@@ -31,17 +32,16 @@ public IQueryable<Capture> GetAllCaptures()
             return _fakeAdminRepo.GetCaptures();
         }
 
-        public IEnumerable<Capture> GetUnnapprovedCaptures()
-        {
-            return _fakeAdminRepo.GetCaptures();
-            ////Eric: is the line above correct? everything after the dot . was missing and I filled it in
-        }
-
         public IEnumerable<Capture> GetUnapprovedCaptures()
         {
-            throw new NotImplementedException();
+            return _fakeAdminRepo.GetAllUnapproved();
```

Comment:
I changed this line with the merge of my branch into develop.  There is no GetAllUnapproved() method in the repository anymore. Instead, GetCaptures() is called to return a collection of all the captures which is then filtered to only return the ones where the approval property type is pending.

Reply:
thanks for letting me know, I'll be sure to make a note of it when it comes time to merge my branch into develop.